modules/RTC/ScreenObtainer.js

```diff
@@ -198,8 +198,11 @@ const ScreenObtainer = {
             logger.info('Using Temasys plugin for desktop sharing');
 
             return obtainWebRTCScreen;
-        } else if (RTCBrowserType.isChrome()) {
-            if (RTCBrowserType.getChromeVersion() < 34) {
+        } else if (RTCBrowserType.isChrome() || RTCBrowserType.isOpera()) {
+            if ((RTCBrowserType.isChrome()
+                    && RTCBrowserType.getChromeVersion() < 34)
+                || (RTCBrowserType.isOpera()
+                        && RTCBrowserType.getOperaVersion() < 34)) {
```

Comment:
Since we have already checked for the browser type let's do the check:
```
(RTCBrowserType.getChromeVersion() || RTCBrowserType.getOperaVersion()) < 34
```

WDYT?